src/constants/ssm-parameter-paths.ts

```diff
@@ -26,6 +26,16 @@ export const SSM_PARAMETER_PATHS: Record<string, SsmParameterPath> = {
     path: "/account/services/private.config.bucket",
     description: "SSM parameter containing the S3 bucket name holding the app's private configuration",
   },
+  AlarmTopic: {
+    path: "/account/services/alarm.topic.arn",
```

Comment:
I'm not entirely convinced that it makes sense to have a default topic for alarms. In my experience multiple teams often share the same AWS account, but not the same topic for alerts (or the same mailing list).

Reply:
Worth noting this will be the default value of the CFN parameter - teams that share accounts would be able to customise it if needed.

Reply:
Yes, that's a good point; I don't think the concern I previously raised should block this PR